src/main/kotlin/lotto/view/SimpleLottoView.kt

```diff
@@ -0,0 +1,30 @@
+package lotto.view
+
+import lotto.view.dto.ConsoleScoreboardDto
+import lotto.view.dto.ConsoleTicketsDto
+
+object SimpleLottoView : LottoView {
+    override fun money(): String? = withInput {
+        println("구입금액을 입력해 주세요.")
+    }
+
+    override fun winningLotto(): String? = withInput {
+        println("지난 주 당첨 번호를 입력해 주세요.")
+    }
+
+    override fun showGeneratedTickets(dto: ConsoleTicketsDto) {
+        println("${dto.count}개를 구매했습니다.")
+        println("${dto.lottos}${System.lineSeparator()}")
+    }
+
+    override fun showScoreboard(dto: ConsoleScoreboardDto) {
+        println(dto.status
+            .joinToString(separator = System.lineSeparator()) { it })
```

Comment:
```suggestion
        println(
            dto.status.joinToString(separator = System.lineSeparator()) { it }
        )
```
파라미터로 전달되는 코드가 길 경우에는 개행해주시면 좀 더 가독성이 좋아집니다 😄